src/frontend/komponenter/Oppgavebenk/FilterSkjemaDatovelger.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { useEffect } from 'react';
+
+import { isValid, parseISO, startOfDay } from 'date-fns';
+
+import { DatePicker, useDatepicker } from '@navikt/ds-react';
+
+import type { IsoDatoString } from '../../utils/dato';
+import { formatterDate } from '../../utils/dato';
+import { Datoformat } from '../../utils/formatter';
+
+interface IProps {
+    value: string | undefined;
+    onDateChange: (dato: IsoDatoString) => void;
+    label: string;
+    visFeilmeldinger: boolean;
+    feilmelding: string | undefined;
+}
+
+const tidligsteRelevanteDato = () => startOfDay(new Date(1900, 0));
+
+const senesteRelevanteDato = () => startOfDay(new Date(2500, 0));
```

Comment:
Når denne (https://github.com/navikt/familie-ba-sak-frontend/pull/2824) er merget kan vi bruke felles-variabler i stedet for å definere opp selv her

Reply:
ser ut som at #2824 merges først, så kan vi oppdatere denne før merge? :)

Reply:
trenger disse å være funksjoner, forresten? kan de bli konstanter?

Reply:
De kan nok være konstanter ja! Kan fikse i den andre PRen